server/.gitignore

```diff
@@ -21,9 +21,6 @@ config/active.dat
 config/logging.json
 /plugins
 
-# Enterprise & products imports files
-channels/imports/imports.go
```

Comment:
I've left the `rm -f` in `make clean`, but I note that transitioning across this PR might be slightly painful. Thoughts on doing a `rm -f` whenever the `Makefile` is invoked if the file is detected?